src/core/draw_tools/atoms/combinedAtom.ts

```diff
@@ -1,11 +1,15 @@
+import _ from 'lodash';
```

Comment:
i believe if you import whole lodash it won't be possible to tree shake it. 
suggest to import only needed stuff like we did before 
`import debounce from 'lodash/debounce';`